### ISSUE TITLE
when I click on launch that coming failed to start no program difined 

### ISSUE THREAD

**Issue by GaziMarzukAhan** [Nilsen84/lunar-client-qt#173]

**Comment by Ekylytra** [comment]
Yeah It is because you have installed (your version) when It is looking for 1.8. To fix this, above the launch with all cosmetics, you can select your corresponding version there.

**Comment by OneSolo63** [comment]
ok the official lunar client made un update that update make the version you have downloaded go to the lunar/version/multiver so when you downloads any version after this update it won't show or it say failed to start no program difined so lunar qt needs to do an update makes the version show when its in multiver 
so please fix this problem
im dying inside ._.

**Comment by 22Toe** [comment]
Yes, that's what I said in many threads.
**Fix: this might be bug from lunar client, that's why they are not updating lunar qt and waiting for lunar to patch it up. If lunar doesn't then devlopers of lunar qt start to fix it.**
